Source/WebCore/editing/MarkupAccumulator.cpp

```diff
@@ -261,6 +261,10 @@ void MarkupAccumulator::serializeNodesWithNamespaces(Node& targetNode, Serialize
                     current = fragment->host();
             }
 
+            ASSERT(current);
```

Comment:
Is there anything above that guarantees `current` won't be null? (I am not very familiar with this code logic so I am just asking since `parentNode()` above seems can return null)

Reply:
In theory, we should never hit a nullptr here because the traversal stays within the subtree given to this function.

Reply:
ah, makes sense.